utils/testmap.py

```
@@ -0,0 +1,142 @@
+# Center the map over central Africa by default
+m = folium.Map(location=[0, 20], zoom_start=3)
+
+image_array, bounds = create_interpolated_overlay(data, resolution=500)
+add_image_overlay(image_array, bounds, m)
+
+# Save the map
+m.save("aqi_mapTEST.html")
+print("Map saved to aqi_map.html")
+
+ 
+print(f"✓ AQI map saved ")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)